docs/source/features/reasoning_outputs.md

```diff
@@ -10,12 +10,12 @@ Reasoning models return an additional `reasoning_content` field in their outputs
 
 vLLM currently supports the following reasoning models:
 
-| Model Series | Parser Name | Structured Output Support | Tool Calling |
-|--------------|-------------|------------------|-------------|
-| [DeepSeek R1 series](https://huggingface.co/collections/deepseek-ai/deepseek-r1-678e1e131c0169c0bc89728d) | `deepseek_r1` | `guided_json`, `guided_regex` | ❌ |
-| [QwQ-32B](https://huggingface.co/Qwen/QwQ-32B) | `deepseek_r1` | `guided_json`, `guided_regex` | ✅ |
-| [IBM Granite 3.2 language models](https://huggingface.co/collections/ibm-granite/granite-32-language-models-67b3bc8c13508f6d064cff9a) | `granite` | ❌ | ❌ |
-| [Qwen3 series](https://huggingface.co/collections/Qwen/qwen3-67dd247413f0e2e4f653967f) | `qwen3` | `guided_json`, `guided_regex` | ✅ |
+| Model Series                                                                                                                          | Parser Name   | Structured Output Support     | Tool Calling |
+| ------------------------------------------------------------------------------------------------------------------------------------- | ------------- | ----------------------------- | ------------ |
+| [DeepSeek R1 series](https://huggingface.co/collections/deepseek-ai/deepseek-r1-678e1e131c0169c0bc89728d)                             | `deepseek_r1` | `guided_json`, `guided_regex` | ❌           |
+| [QwQ-32B](https://huggingface.co/Qwen/QwQ-32B)                                                                                        | `deepseek_r1` | `guided_json`, `guided_regex` | ✅           |
+| [IBM Granite 3.2 language models](https://huggingface.co/collections/ibm-granite/granite-32-language-models-67b3bc8c13508f6d064cff9a) | `granite`     | ❌                            | ❌           |
+| [Qwen3 series](https://huggingface.co/collections/Qwen/qwen3-67dd247413f0e2e4f653967f)                                                | `qwen3`       | `guided_json`, `guided_regex` | ✅           |
```

Comment:
Is this just whitespace changes? If so, please remove it to keep the PR more focused

Reply:
my bad, will disable formatter for this.